src/tools/pub/cmd/pub-get.md

```diff
@@ -56,12 +56,14 @@ get the latest versions of all dependencies.
 ## Package resolution
 
 By default, pub creates a `package_config.json` file
-in the `.dart_tool/` directory that maps from package names to location URIs.
+in the `.dart_tool/` directory that maps from
+package names to location URIs.
 Before the `package_config.json` file, 
-pub used to create a `.packages` file in the root directory.
+pub used to create a `.packages` file in the root directory,
+but that file is now deprecated.
 
 {{site.alert.note}}
-  Don't check the generated `.packages` file
+  Don't check the generated `package_config.json` file
   or the `.dart_tool/` directory into your repo;
```

Comment:
The `package_config.json` file is part of the  `.dart_tool/` directory, so I don't think this is needed:

```suggestion
  Don't check the generated `.dart_tool/` directory into your repo;
```